### PR TITLE
Setup and fix some tests

### PR DESCRIPTION
**Pull request by jbarnoud**
We are not going to have everything unit tested in one go, but let's at least start and have the tests running on travis.

### CODE REVIEW

**Comment by jbarnoud** [comment]
The PR is ready for review. Sorry if it produced a lot of notifications...

The test fails at the moment because they already account for #60 and #61.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - "3.4"
+    - "3.5"
+    - "3.5-dev"
+    - "3.6"
+    - "3.6-dev"
+    - "3.7-dev"
+matrix:
+    allow_failures:
+        - python: "2.7"
+        - python: "3.4"
```
Comment (pckroon) [inline .travis.yml]: 3.4 is not an allowed failure, and it's errors should be fixed.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    apt:
+        packages:
+            - dssp
+env:
+    - VERMOUTH_TEST_DSSP=mkdssp
+install:
+    - pip install pylint
+    - pip install --upgrade .
+script:
+    - pytest -vv vermouth
```
Comment (pckroon) [inline .travis.yml]: -vv? Is that nescessary?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+after_script:
+    - pylint vermouth
+    - pylint bin/martinize2
```
Comment (pckroon) [inline .travis.yml]: You could argue that this should be part of the test, and not just a nice output to see. Either way, we'll probably need a custom .pylintrc. 
I guess pytest has a pylint plugin, to wrap it all in one.
Also, do we want to look at test coverage?

File: .travis.yml (hunk 2 above)
Comment (jbarnoud) [reply .travis.yml]: The main point is to display the whole difference when an assert fails. My experience is that you will likely rerun the failing tests with -vv so having it from the start saves some time.

File: .travis.yml (hunk 3 above)
Comment (jbarnoud) [reply .travis.yml]: There are way too many messages at the moment. We need a .pylintrc AND a lot of fixing before we can use it as a test.

Comment (pckroon) [reply .travis.yml]: Let's keep it in mind for the future. 
And yes, if I recall correctly, PyLint is rather... How do I phrase this... Strict.

**Comment by jbarnoud** [comment]
Should be fixed now.

**Comment by jbarnoud** [comment]
Also, I suggest we add the coverage latter. First we have to choose what service to use to display the coverage, but also we do not have enough tests yet for the coverage to be meaningful.

**Comment by pckroon** [comment]
Nice work, thanks :)

**Pull request merged**